File: ios/Classes/model/CallCreateAd.swift
import Foundation
import AdropAds

struct CallCreateAd {
    var unitId: String
    var requestId: String
    
    init(unitId: String, requestId: String) {
        self.unitId = unitId
        self.requestId = requestId
    }
    
    init(encoding: [String: Any?]?) {
        self.unitId = encoding?["unitId"] as? String ?? ""
        self.requestId = encoding?["requestId"] as? String ?? ""
    }
}

File: android/src/main/kotlin/io/adrop/adrop_ads/model/CallCreateAd.kt
package io.adrop.adrop_ads.model


data class CallCreateAd(
    var unitId: String,
    var requestId: String = "",
    var useCustomClick: Boolean = false
) {
    constructor(encoding: Map<String, Any?>?) : this(
        unitId = encoding?.get("unitId") as? String ?: "",
        requestId = encoding?.get("requestId") as? String ?: "",
        useCustomClick = encoding?.get("useCustomClick") as? Boolean ?: false
    )
}